shell/pages/explorer/resource/detail/projectsecret.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+import DetailPage from '@shell/components/Resource/Detail/Page.vue';
+import TitleBar from '@shell/components/Resource/Detail/TitleBar/index.vue';
+import { useDefaultTitleBarProps } from '@shell/components/Resource/Detail/TitleBar/composables';
+import Metadata from '@shell/components/Resource/Detail/Metadata/index.vue';
+import { useBasicMetadata } from '@shell/components/Resource/Detail/Metadata/composables';
+import { MANAGEMENT, SECRET } from '@shell/config/types';
+import { useFetchResourceWithId, useResourceIdentifiers } from '@shell/composables/resources';
+import ResourceTabs from '@shell/components/form/ResourceTabs/index.vue';
+import SecretDataTab from '@shell/components/Resource/Detail/ResourceTabs/SecretDataTab/index.vue';
+import KnownHostsTab from '@shell/components/Resource/Detail/ResourceTabs/KnownHostsTab/index.vue';
+import { useGetKnownHostsTabProps } from '@shell/components/Resource/Detail/ResourceTabs/KnownHostsTab/composables';
+import { useSecretDataTabDefaultProps } from '@shell/components/Resource/Detail/ResourceTabs/SecretDataTab/composeables';
+import { useProjectSecretIdentifyingInformation } from '@shell/components/Resource/Detail/Metadata/IdentifyingInformation/composable';
+import { useStore } from 'vuex';
+
+const store = useStore();
+
+const { id, schema } = useResourceIdentifiers(SECRET);
+const secret = await useFetchResourceWithId(SECRET, id);
+
+// We need to request the project associated with the secret to display the metadata for it.
+// We probably shouldn't use findall but there appeared to be something odd going on with the model not finding the id when I used management/find.
+await store.dispatch('management/findAll', { type: MANAGEMENT.PROJECT });
```

Comment:
I'm not sure why we would need to override the store/resource when you can specify it right here for the new detail page for the new resource that was created.

We could forward the old logic to the new detail pages but it's much easier for me to me to go to the page definition to figure out how things are called and defined instead of going through multiple parents/children and mixins to figure out where overriden stores and resources may or may not be used.